pluto-message-ingestion/file-config.js

```diff
@@ -0,0 +1,53 @@
+const fs = require('fs');
+const AWS = require('aws-sdk');
+const parseHocon = require('hoconjs/build/hoconjs');
+
+const EnvironmentConfig = require('./environment-config');
+
+const FILE_NAME = 'media-atom-maker.private.conf';
+
+class FileConfig {
+  static read() {
+    if (EnvironmentConfig.isDev) {
+      return FileConfig._readFromDisk();
```

Comment:
Yep, could do. Only reason not to was to avoid a network call as we already have the file on disk... happy to remove though as its simpler to do the same thing in all environments.